packit_service/constants.py

```diff
@@ -28,6 +17,17 @@
 DOCS_VALIDATE_CONFIG = f"{DOCS_URL}/cli/config/validate"
 DOCS_VALIDATE_HOOKS = "https://packit.dev/posts/pre-commit-hooks#validate-config"
 
+HELP_COMMENT_PROG = "/packit | /packit-stg"
+HELP_COMMENT_PROG_FEDORA_CI = "/packit-ci | /packit-ci-stg"
+HELP_COMMENT_DESCRIPTION = ""
+HELP_COMMENT_EPILOG = (
+    f"Documentation: {DOCS_URL}\n"
+    "Contact:\n"
+    "     Email:    hello@packit.dev\n"
+    "     Matrix:   #packit:fedora.im\n"
+    "     Mastodon: @packit@fosstodon.org"
+)
+
```

Comment:
Could you fix up this ( 5836e60a428d039e57964ae1765541150a4954fe ) commit into this one ( 8bf7bb00bdd63bb2ad833e23fe41cab72f061571 )? I don't think it makes much sense to have it separate.